src/reducers/index.js

```diff
@@ -5,12 +5,14 @@ import registrationReducer from "./registrationReducer";
 import userSettingsReducer from "./userSettingsReducer";
 import profileReducer from "./profileReducer";
 import article from "./articlesReducers";
+import tagReducer from "./tagSearchReducer";
 
 export default history => combineReducers({
   router: connectRouter(history),
   registrationReducer,
   loginReducer,
   userSettingsReducer,
   profileReducer,
-  article
+  article,
+  tagReducer
```

Comment:
We have been told to remove the suffix Reducer from these keys.

Reply:
@bettblake08 thanks for spotting that let me work on it.